packages/react-native/React/Base/RCTConvert.mm

```diff
@@ -377,7 +377,7 @@ + (NSLocale *)NSLocale:(id)json
 
 + (NSLineBreakStrategy)NSLineBreakStrategy:(id)json RCT_DYNAMIC
 {
-  if (@available(iOS 14.0, macOS 11.0, *)) { // [macOS]
+  if (@available(iOS 14.0, *)) {
```

Comment:
... this likely answers my previous question.

Reply:
Meta supports iOS 13.4 as a minimum, and we keep that. For 0.76+, it will be bumped to iOS 15.1. At that point, I expect a similar commit upstream that removes some of these availability checks.